sdk/src/client/stronghold/mod.rs

```diff
@@ -286,7 +287,7 @@ impl StrongholdAdapter {
     /// data, provide a list of keys in `keys_to_re_encrypt`, as we have no way to list and iterate over every
     /// key-value in the Stronghold store - we'll attempt on the ones provided instead. Set it to `None` to skip
     /// re-encryption.
-    pub async fn change_password(&self, new_password: &str) -> Result<(), Error> {
+    pub async fn change_password(&self, new_password: Password) -> Result<(), Error> {
```

Comment:
Not Into?

Reply:
outdated ... can be closed ... I think we made sure that `.into()` is called whenever we have an `impl Into<Password>`